frontend/src/features/mapSideBar/SidebarContents/AssociatedLandForm.tsx

```diff
@@ -170,7 +170,7 @@ const Form: React.FC<IAssociatedLandForm> = ({
   handlePidChange,
   handlePinChange,
   formikRef,
-  isAdmin,
+  isSres,
```

Comment:
Technically the rule is they must have the `adminProperties` claim rather than the Role name.  As the roles can change in the future.